service/src/app/models.py

```
@@ -15,8 +17,128 @@ class Base(DeclarativeBase):
 
 
+class PersonOrganisationalUnitStaffAssociation(Base):
+    __tablename__ = "persons_organisational_units_staff_association"
+    person_id: Mapped[UUID] = mapped_column(
+        ForeignKey("persons.person_id"), primary_key=True
+    )
+    organisational_unit_id: Mapped[UUID] = mapped_column(
+        ForeignKey("organisational_units.organisational_unit_id"), primary_key=True
+    )
+    period: Mapped[Optional[DateTimeTZRange]] = mapped_column(TSTZRANGE)
+    person: Mapped[List[Person]] = relationship(
```

Ruff (0.15.11)

[error] Undefined name `Person`

(F821)